src/loinclib/attic/tmp/tmp-loinclib-graph-enums.py

```diff
@@ -2,7 +2,7 @@
 
 import loinclib
 
-print(loinclib.NodeType.type_for_identifier("4-4"))
+print(loinclib.NodeHandler.type_for_identifier("4-4"))
```

Comment:
Will this not result in an `AttributeError`? I don't see any changes to `loinclib/__init__.py` in this PR, and in my local, I don't see `NodeHandler` as an import there. Perhaps it comes with one of the `*` imports (not a big dealy, but we should try to avoid):

```py
from .graph import *
from .snomed_loader import *
```